build/lib/helm_copy.sh

```diff
@@ -34,5 +34,5 @@ source "${SCRIPT_ROOT}/common.sh"
 # Copy
 #
 mkdir -p ${DEST_DIR}
-cp -r ${SOURCE_DIR} ${DEST_DIR}
+rsync -a --exclude .git "${SOURCE_DIR}" "${DEST_DIR}"
```

Comment:
kind of odd to me to use rsync for a copy that is totally not remote. I guess the only concern is what you mentioned about it being in the build tooling image and looking at the tests that were kicked off from there, none of them may create a helm chart.

Reply:
oh, yeh, I have to scroll through the builds, there are definitely some helm builds in there

Reply:
worked https://prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp.s3.us-west-2.amazonaws.com/pr-logs/pull/aws_eks-anywhere-build-tooling/790/hello-eks-anywhere-tooling-presubmit/1524438268534329344/build-container-build-log.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAUTLHWUYSHGSYOX4K%2F20220511%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20220511T172349Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEFkaCXVzLXdlc3QtMiJHMEUCIQDqfoCDuyYw1iF5Fn6AAwaXGCPsf9N%2BZ7zL2rZbuiJYOgIgS4VTAHDn4Gs%2FmPwfQ98ayMcohe7GHdxQI7ZvsxlPHNAq8AQIMhABGgwzMTY0MzQ0NTgxNDgiDArzdPqtYsHlsh0jyirNBBrUHVTNH%2FTnwgaJ6uKKN1DqEi3B8vS1gQ3LylzSpxRxBAM5%2Bs8YzcywSnW8HACF25nixFEAfM6I5WzUZtCJhLbF78orR35dXOgORd0H34GiCUiGaPaO%2B40OqQA%2BDj8Wn2BLTqETnK%2BWgwacLeZv1RmgkQjfzm4wucOoJgBadQ2wZVbSTpaFW9z6FxXFcJFKC68goHGekAbsOF8YptSyDU4Z%2BVrqJn7cLLonPg7Bin53NgiYtpOjIheBUAjJRNjOq0fRKCW5UK81FRloKI0EkPExIpJZ4BXNf6N78ZjVbSoEqBjq4smL6%2BuUXk5xzVZQvdQlg4STPZt8iap6Uo1XYeJPBmfVaEYS8e0S8gbbG3hJF9TS%2B8LbgSZRyqVfWBO9IeIeGrfHNkOMkRP4ZsCAzQV%2FaYNEpfU9W4PbdccqGoTYdtQOTuuRec0G%2FkSjaxQzNpyyxMFV4ppz%2BSVTyvsH2lo4EKBIJJPfAk0HcDJAl0pMT5902M1wIv3kvbr4BjwJ6onpX1tFZ%2Fen2h5SzFg2dhTyZQXuFCXq6SYbLcMNVdT54XB7oIxCg6nCa40Tsn1799UrjwZYTMgRAdWzOOmKqDw%2BB8nyFv34KylX2TmikUKHfAuqh4hatvX2GoOfim9YyK09hN9n3j3qOM3C0alT7WuxfO%2B52pscG70uGe%2B3MTeKgx%2B2EJrHtBW4izf0mQvJOwi5U98uXlEw6nNFKI3meR80aakkXF3Zhtf1MmSBmx9rEGu7iuzl7xm34KHERs9QOD0IDzZE55CnHBU9xEkwhNfvkwY6mgGz8ONnxU4OV1ftyp6%2B5OSfPNiqv%2FdiLQ85Bui2cV1CMdRowJkMebcBqb%2BeewctHLza8%2BE%2BmxrFLGNG3zLX0824q4RkwU6HmqllxYEoQctPIhFu1usCfyHWlaRWuccQlIEIWlqeWR71Xzj%2FAXF7Sf1owAr9F19uxt4KLLHkMJ%2FGGL33%2FQRb6X0JNPoT%2B7DeaC8oex24%2FqyUKWMA&X-Amz-SignedHeaders=host&X-Amz-Signature=2499aed8b5d881040b5a0b8ef4eaf2480439c8af331e72136cef672f9df85f05

Reply:
> kind of odd to me to use rsync for a copy that is totally not remote. I guess the only concern is what you mentioned about it being in the build tooling image and looking at the tests that were kicked off from there, none of them may create a helm chart.

Rsync is great for local copies too! The man page even says:
> rsync - a fast, versatile, remote (and local) file-copying tool

Hehe, but more seriously, rsync can totally be used locally. It's nice for moving really large directories around, since it compares before moving, it can save a lot of time. It also has handy features like a progress bar (if you need it) and the excludes, which I'm using to good effect here. :)